gpkit/geometric_program.py

```diff
@@ -174,11 +193,10 @@ def _get_solver(solver):
                    ((time() - tic) / soltime * 100))
             tic = time()
 
-        np.ravel(solver_out['primal'])
         self.check_solution(result["cost"],
                             np.ravel(solver_out['primal']),
-                            nu=result["sensitivities"]["monomials"],
-                            la=result["sensitivities"]["posynomials"])
+                            nu=result["sensitivities"]["nu"],
+                            la=result["sensitivities"]["lambda"])
```

Comment:
should it be `"la"` instead of `"lambda"` to be consistent with the argument of `check_solution`, the key in `solver_out`, and to avoid potential variable names of `lambda`?